Adding additional 2FA on SSH troubleshooting and sample automation script (#7558)
magento/devdocs: src/cloud/project/project-enable-mfa-enforcement.md

    remote: counting objects: 22, done.
    Receiving objects: 100% (22/22), 82.42 KiB | 16.48 MiB/s, done.
    ```

## Troubleshooting

> 

Reviewer: Need an intro paragraph for troubleshooting:     Use the following information to resolve SSH connection requests failures due to authentication errors like `access requires MFA` or `permission denied`.

   magento-cloud ssh-cert:load
   ```

### Unable to authenticate to projects that do not have MFA enabled

> If you authenticate to a project with MFA enabled, you may receive the following error on other projects:

Reviewer: ```suggestion
If you authenticate to a project with MFA enabled, you might receive the following error when connecting to other projects that do not require MFA:
```

   ```bash
   ssh abcdef7uyxabce-master-7rqtabc--mymagento@ssh.us-3.magento.cloud
   abcdef7uyxabce-master-7rqtabc--mymagento@ssh.us-3.magento.cloud: Permission denied (publickey).
   ```

> During the SSH certificate generation, the Magento Cloud CLI adds an additional ssh certificate to your local environment. If your previous key was not added to the SSH configuration, it will no longer be used as a default.

Reviewer: ```suggestion
During the SSH certificate generation, the Magento Cloud CLI adds an additional SSH certificate to your local environment. The certificate will be used by default if the SSH key for project access is not included in your local SSH configuration.
```

{:.procedure}

> To add your SSH key to your local configuration:

Reviewer: ```suggestion
To add your SSH key to the local configuration:
```

1. Create the `config` file if it does not exists.

    ```bash
    touch ~/.ssh/config
    ```

1. Add an `IdentityFile` configuration.

    ```yaml
   Host *
     IdentityFile /Users/<user-name>/.ssh/id_rsa
    ```

   {:.bs-callout-info}

>    You can specify multiple SSH keys by adding multiple IdentityFile entries to your configuration.

Reviewer: ```suggestion
   You can specify multiple SSH keys by adding multiple `IdentityFile` entries to your configuration.
```

1. Close your terminal session. The configuration will be used for all sessions going forward.

## Automatically generate certificate on login

> Configurations can be added to your local environment to automatically generate an SSH certificate when you authenticate to the Magento Cloud CLI.

Reviewer: ```suggestion
You can configure your local environment to automatically generate an SSH certificate when you authenticate to the Magento Cloud CLI.
```

{:.procedure}

> To add SSH certificate auto generation to your configuration:

Reviewer: ```suggestion
To add SSH certificate auto generation to your Magento Cloud CLI configuration:
```
